### PR TITLE
fix: eth transaction list displaying partly

### PR DESCRIPTION
**Pull request by martinkaintas**
missed in #2618  as it was done on develop branch

### CODE REVIEW

File: src/protocols/ethereum/views/AccountDetails.vue [hunk 1]
```diff
@@ -16,13 +19,19 @@
...
+  ref,
+  watch,
+} from 'vue';
 import { IonContent, IonPage } from '@ionic/vue';
 import { PROTOCOL_VIEW_ACCOUNT_DETAILS } from '@/constants';
 import { ROUTE_ACCOUNT_DETAILS, ROUTE_ACCOUNT_DETAILS_ASSETS } from '@/popup/router/routeNames';
 
 import AccountDetailsBase from '@/popup/components/AccountDetailsBase.vue';
 import AccountDetailsNavigation from '@/popup/components/AccountDetailsNavigation.vue';
+import { IonicLifecycleStatus } from '@/types';
```
Comment (peronczyk) [inline src/protocols/ethereum/views/AccountDetails.vue:34]: Please put this line before the constants import.

Comment (martinkaintas) [reply src/protocols/ethereum/views/AccountDetails.vue:34]: Damn auto-imports :S

File: src/protocols/ethereum/views/AccountDetails.vue [hunk 2]
```diff
@@ -32,10 +41,24 @@ export default defineComponent({
     IonContent,
     AccountDetailsNavigation,
   },
-  setup() {
+  props: {
+    ionicLifecycleStatus: { type: String as PropType<IonicLifecycleStatus>, default: null },
```
Comment (peronczyk) [inline src/protocols/ethereum/views/AccountDetails.vue:45]: What do you think about replacing this `ionicLifecycleStatus` with a bool prop? Is this possible? I see we have 4 possible values for this, but isn't 2 of them truthy and 2 falsy?

Or maybe having 2 separate bool props: `pageDidEnter` and `pageWillEnter`.

The benefit of boolean prop is that we can avoid importing and using the `IonicLifecycleStatus` type. Also we could stop using the `'didEnter'` string values. Instead the prop could be used directly in the template to display or hide the elements.

**Review by peronczyk — CHANGES_REQUESTED** [review]

File: src/protocols/ethereum/views/AccountDetails.vue [hunk 3]
```diff
@@ -16,7 +19,9 @@
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue';
+import {
+  defineComponent,
+} from 'vue';
```
Comment (peronczyk) [inline src/protocols/ethereum/views/AccountDetails.vue:24]: No need for this change.

**Comment by Liubov-crypto** [comment]
Can't get my ETH Tx list, probably ethereum testnet issue: 

уер

**Comment by Liubov-crypto** [comment]
Finally I managed to get it, in general LGTM 

But ETH transactions are visible in LINK TXs list and vice versa:
tok det

err1

**Pull request merged**